x-pack/plugins/apm/public/components/shared/TraceLink.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+// @ts-ignore
+import { legacyEncodeURIComponent, RelativeLink } from '../../utils/url';
+
+interface Props {
+  name: string;
+  serviceName?: string;
+  transactionType?: string;
+  transactionId?: string;
+  traceId?: string;
+}
+
+export const TraceLink: React.SFC<Props> = ({
```

Comment:
Is `TraceLink` used anywhere?

Reply:
WIP :)

Reply:
oh this is actually WIP for my other ticket, but got committed here as i switched back and forth today, i'll move it